### PR TITLE
Fix gemgift spacing

### PR DESCRIPTION
**Pull request by vincecampanale**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)
Included intro in gems message to fix spacing issues with translations. 

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes issue #8283 

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
Added the content of `privateMessageGiftIntro` to `privateMessageGiftGemsMessage` and adjusted the relevant tests to ensure the build passes. Also made some minor syntax changes to previous commit to pass tests. 

[//]: # (Put User ID in here - found in Settings -> API)

----
UUID: 08c2b5db-70bd-4677-8a95-d84cdfec2afb

### CODE REVIEW

File: website/common/locales/en/groups.json [hunk 1]
```diff
@@ -114,8 +114,8 @@
     "gemAmountRequired": "A number of gems is required",
     "notAuthorizedToSendMessageToThisUser": "Can't send message to this user.",
     "privateMessageGiftIntro": "Hello <%= receiverName %>, <%= senderName %> has sent you ",
```
Comment (lynxlynxlynx) [inline website/common/locales/en/groups.json]: privateMessageGiftIntro can go away now.

Comment (vincecampanale) [reply website/common/locales/en/groups.json]: @khipkin mentioned that it may be used for the subscription gift.

Comment (lynxlynxlynx) [reply website/common/locales/en/groups.json]: You removed all the users — it is only used for gifting gems and its tests.

Comment (khipkin) [reply website/common/locales/en/groups.json]: It seems like `"privateMessageGiftSubscriptionMessage"` is no longer used in the code base. It is probably okay to delete that.

File: website/common/locales/en/groups.json [hunk 2]
```diff
@@ -113,9 +113,7 @@
     "toUserIDRequired": "A User ID is required",
     "gemAmountRequired": "A number of gems is required",
     "notAuthorizedToSendMessageToThisUser": "Can't send message to this user.",
-    "privateMessageGiftIntro": "Hello <%= receiverName %>, <%= senderName %> has sent you ",
-    "privateMessageGiftGemsMessage": "<%= gemAmount %> gems!",
-    "privateMessageGiftSubscriptionMessage": "<%= numberOfMonths %> months of subscription! ",
```
Comment (paglias) [inline website/common/locales/en/groups.json:118]: this string has been removed because it wasn't used?

**Comment by paglias** [comment]
Apart from comment, this looks good!

**Comment by vincecampanale** [comment]
@paglias Thanks! Yes, I removed `privateMessageGiftIntro` and `privateMessageGiftSubscription` because neither is used anywhere in production. If you guys want, I can add back the subscription gift with the same format as the gem gift. Let me know :) And thanks for letting me contribute!

**Comment by paglias** [comment]
@vincecampanale merged! thanks for the help :)

**Pull request merged**